lib/any.js

```diff
@@ -683,6 +683,21 @@ module.exports = internals.Any = class {
             }
         }
 
+        if (this._settings) {
+
+            const optionSet = ['abortEarly', 'convert', 'allowUnknown', 'skipFunctions', 'stripUnknown', 'language', 'presence', 'context', 'noDefaults'];
+            if (optionSet.some((option) => this._settings.hasOwnProperty(option))) {
```

Comment:
Can you expand on which properties you are filtering with this specifically ? Maybe they shouldn't be there in the 1st place.